namespaces/live.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-dev/resources/claim-criminal-injuries-application-queue-polices.tf

```
@@ -82,4 +82,56 @@ resource "kubernetes_secret" "app-service-sqs-secret" {
     access_key_id     = aws_iam_access_key.app_service.id
     secret_access_key = aws_iam_access_key.app_service.secret
   }
+}
+
+data "aws_iam_policy_document" "redrive_service_access" {
+  statement {
+    sid = "AllowRedriveServiceToReadFromAppDLQ"
+    actions = [
+      "sqs:DeleteMessage",
+      "sqs:ReceiveMessage",
+      "sqs:GetQueueAttributes"
+    ]
+    resources = [
+      module.claim-criminal-injuries-application-dlq.sqs_arn
+    ]
+  }
+  
+  statement {
+    sid = "AllowRedriveServiceToSendToAppQueue"
+    actions = [
+      "sqs:SendMessage"
+    ]
+    resources = [
+      module.claim-criminal-injuries-application-queue.sqs_arn
+    ]
+  }
+}
+
+resource "aws_iam_user" "redrive_service" {
+  name = "app_service-${random_id.id.hex}"
```

```suggestion
  name = "cp-sqs-redrive-${random_id.id.hex}"
```

This is to match how the other SQS IAM users are named.

---

Thanks Jake, just spotted this myself! Updated.